terraform/modules/azure-jenkinsinfra-storage/variables.tf

```
@@ -0,0 +1,28 @@
+# Required variables
+variable "service_fqdn" {
+  type = string
+}
+
+variable "storage_service_principal_ids" {
+  type = list(string)
+}
+
+variable "storage_service_principal_end_date" {
+  type = string
+}
+
+variable "storage_file_share_ids" {
+  type    = list(string)
+  default = []
+}
+
+# Optional variables
+variable "default_tags" {
+  type    = map(string)
+  default = {}
+}
+
+variable "storage_active_directory_url" {
+  type    = string
+  default = "https://github.com/jenkins-infra/azure"
```

```suggestion
  default = ""
```

or 

```suggestion
  default = null
```

as we can call this from different repositories